.travis.yml

```diff
@@ -21,47 +21,50 @@ env:
    global:
    - PROJECT="JSQDataSourcesKit.xcodeproj"
    - SCHEME="JSQDataSourcesKit"
-   - IOS_SDK=iphonesimulator13.0
-   - TVOS_SDK=appletvsimulator13.0
+   - IOS_SDK=iphonesimulator13.2
+   - TVOS_SDK=appletvsimulator13.2
 
 before_install:
   - pip3 install proselint
-  - gem update --system
-  - gem install bundler
+  - make setup
 
 jobs:
   include:
     - &test
       stage: unit tests
-      name: iOS 11.1
-      env: DESTINATION="OS=11.1,name=iPhone X" SDK="$IOS_SDK"
+      name: iOS 11.4
+      env: DESTINATION="OS=11.4,name=iPhone X" SDK="$IOS_SDK"
```

Comment:
can we leave this at `11.1` or is that no longer available?

Reply:
It should be.

Sorry about the versioning.  I was looking at a previous, passing revision in a first attempt to solve the failure.  Will keep your preference in mind moving forward 😉